src/iuse_actor.h

```diff
@@ -1179,6 +1179,8 @@ class effect_on_conditons_actor : public iuse_actor
 
         ~effect_on_conditons_actor() override = default;
         void load( const JsonObject &obj ) override;
+        /**does the item requires to be wielded to be activable*/
+        bool need_wielding = false;
```

Comment:
to be documented in https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_INFO.md?plain=1#L4453

Reply:
Need to get it working first lol.